tests/unit/test_blob.py

```diff
@@ -1982,7 +1983,7 @@ def test__do_multipart_upload_bad_size(self):
 
         with self.assertRaises(ValueError) as exc_info:
             blob._do_multipart_upload(
-                None, stream, None, size, None, None, None, None, None, None
+                client, stream, None, size, None, None, None, None, None, None
             )
```

Comment:
Given that the passed-in `client` is not used by `Blob._do_multipart_upload` before the `ValueError` is raised, this change is unnecessary / distracting.

We do need extra test cases that verify that the passed-in `client` is used in preference to `self._client` in both `Blob._do_multipart_upload` and `Blob._initiate_resumable_upload`.